composer.json

```diff
@@ -2,9 +2,8 @@
     "name": "atoum/ruler-extension",
     "require": {
         "atoum/atoum": ">=1.0,<3.0",
-        "hoa/ruler": ">=1.15.11.09,<2.0",
-        "hoa/regex": ">=0.15.05.29,<1.0",
-        "hoa/core": ">=2.14.12.10,<3.0"
+        "hoa/ruler": ">=2.16.01.11,<3.0",
+        "hoa/stream": ">=0.16.01.11,<1.0"
```

Comment:
why do we need `hoa/stream` here ? If it's required for `hoa/ruler` then `hoa/ruler` should declare this dependency itself